terraform/deployments/tfc-configuration/gcp-projects.tf

```diff
@@ -0,0 +1,32 @@
+module "gcp-ga4-analytics-352613" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.10.0"
+
+  organization        = var.organization
+  workspace_name      = "gcp-ga4-analytics-352613"
+  workspace_desc      = "BigQuery infrastructure for Data Insight & Analytics"
+  workspace_tags      = ["production", "ga4-analytics-352613", "gcp"]
+  terraform_version   = var.terraform_version
+  execution_mode      = "remote"
+  working_directory   = "/terraform/deployments/ga4-analytics-352613/"
+  trigger_patterns    = ["/terraform/deployments/ga4-analytics-352613/**/*"]
+  global_remote_state = true
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
+    "GOV.UK Non-Production" = "write"
```

Comment:
I consciously made that decision because I'm not in GOV.UK Production but wanted to be able to initiate runs. Adding Non-Production is a lazy workaround but t feels like adding me to Production is too much and creating a new team for me is also too much, what do you think?

Reply:
Thinking more about it, I should solve the problem and not do a bad workaround. I will ask for a new team to be created, GCP Data Production or something like that